### ISSUE TITLE
Any changes to DMControl source code?

### ISSUE THREAD

**Issue by Manchery** [nicklashansen/dmcontrol-generalization-benchmark#3]
Hi, great work!

I've noticed that `this benchmark differs from the original implementation of Distracting Control Suite.`

But how about the DMC source code here compared with official repo? Any changes?

I think some much more highlighted and detailed notifications on README are neccesarry and welcomed.

**Comment by nicklashansen** [comment]
Hi,

Thank you for your feedback. I have added additional information on our Distracting Control Suite implementation in the test environments section of the README. Our changes can be summarized as follows: (1) we evaluate at a different set of intensities (and number of videos) that more closely matches performance of current algorithms; (2) we reduce randomization update frequency by a factor of 2 to account for frame skip (action repeat); (3) all Tensorflow dependencies have been replaced by PyTorch.

We make no changes to the DMC and dmc2gym source code besides adding support for our proposed test environments (primarily implemented [here](https://github.com/nicklashansen/dmcontrol-generalization-benchmark/blob/main/src/env/dm_control/dm_control/suite/common/settings.py)).

I hope that you find the repository useful. I will leave this issue open for a while in case you have additional feedback.
